one-point-oh.md

```diff
@@ -0,0 +1,155 @@
+# 1.0 plan and priorities
+
+We don't have time for big changes or big features (dev time + time to marinate). We know vaguely where we're going (see other design PRs).
+
+High level priorities:
+
+- Stability and polish
+  - bugs, performance, glitches, etc.
+- User experience
+  - Goal: KCL is easy to read and comprehend
+  - Feedback is useful (though perhaps not pretty)
+    - IDE experience
+    - Error messages (parser in particular)
+- Supporting core new features in engine, as required
+- Future compatibility
+- Not making tech debt worse
+  - And ideally a little bit better
+
+Related:
+
+- [Modelling app roadmap](https://github.com/orgs/KittyCAD/projects/35)
+- [1.0 milestone](https://github.com/KittyCAD/modeling-app/milestone/2)
+- [CAD primitives roadmap](https://github.com/KittyCAD/modeling-app/issues/729)
+
+Priorities in **bold**, dependencies in *italics*, task size in t-shirt sizes (taking into account breaking change management; m-l means uncertainty about which, not 'somewhere in between').
+
+## As required
+
+### Bug fixing
+
+**P0**:
+
+- high priority bugs as they emerge
+- plan to seek out and address lower-priority bugs during feature freeze
+
+
+### Non-language feature work
+
+Priority follows from other team's priorities.
+
+New or more flexible std lib functions and code mods are expected as side effects of other work.
+
+New features which might require significant work:
+
+- [Multiple profiles in Sketches](https://github.com/KittyCAD/modeling-app/issues/1876)
+- Assemblies?
+
+
+### Pre-paying technical debt
+
+**P1**:
+
+- Don't make things worse by adding hacks, especially around the API of KCL
+
+**P3**:
+
+- Supplying user-side IDs to engine
+- code mod API
+
+
+## Language priorities
```

Comment:
I'll add it in. I'll need to think a bit about how it can work, etc. My initial impression is that it will be a pretty large amount of work.

Are we aiming to get assemblies into 1.0? That was one thing I was unsure about for the feature work

Reply:
yes assemblies are required for v1, honestly i think we are like 50% of the way there for cache, we already have the hashes, we would need to only re-execute the objects that changed, there is already an api for object delete.

Reply:
I added work items for assemblies and caching to the doc, needs elaboration though